README.rst

```diff
@@ -24,24 +24,27 @@ We use Git tags to pin a stack version and release a stack image to Docker hub.
 
 Versioning is done in the following manner:
 
-- major stack versions are marked with ``<scrapy version>[-py3]`` tag.
+- major stack versions are marked with ``<scrapy version>[-pyX]`` tag.
```

Comment:
I think this change only makes sense if `-py37` and similar can happen. But I have not seen such a tag so far, except for your mention below.

Reply:
These tags do exist:
- https://github.com/scrapinghub/scrapinghub-stack-scrapy/releases/tag/1.7-py38-20191018
- https://github.com/scrapinghub/scrapinghub-stack-scrapy/releases/tag/1.6-py37

Reply:
Oh, nice!